resources/Dockerfile

```diff
@@ -65,6 +65,19 @@ RUN if [ "$TARGETARCH" = "aarch64" ]; then \
         && cp libfdt/libfdt.a /usr/lib/aarch64-linux-musl \
         && rm -rf /dtc;	fi
 
+RUN if [ "$TARGETARCH" = "aarch64" ]; then \
+        # Build and install the musl-gcc version of openssl. This is to ensure
+        # the unit and integration test binary (both use the ssh2 crate which needs openssl)
+        # can be build and linked using musl toolchain.
+        ln -s /usr/include/aarch64-linux-gnu/asm /usr/include/aarch64-linux-musl/asm \
+        && ln -s /usr/include/asm-generic /usr/include/aarch64-linux-musl/asm-generic \
+        && ln -s /usr/include/linux /usr/include/aarch64-linux-musl/linux \
+        && mkdir /openssl-musl && wget https://github.com/openssl/openssl/archive/OpenSSL_1_1_1f.tar.gz \
```

Comment:
Wondering if we need to pin the specific version of openssl.

Reply:
Also do we need to update the version of Dockerfile again? This modification is also AArch64 only, even if we keep the version the same as before (i.e. v4), nothing will be broken.